### PR TITLE
Adding: Capacitor Regulator

### PR DESCRIPTION
**Pull request by Masterdiekillsu**
Adds an attachment for laser guns to increase mag size on a universal level.

### CODE REVIEW

File: Items/04_00_WeaponAttachments.txt [hunk 1]
```diff
@@ -350,6 +350,14 @@ more effeciently. This attachment can only be put on laser weapons.
 
     Weapon Damage:  +10% (Drop the remainder) or +5, whichever is higher.
 
+== Capacitor Rgulator $1500 ==
```
Comment (bleehu) [inline Items/04_00_WeaponAttachments.txt:353]: SP: "Regulator"

**Review by Turtlelord26 — COMMENTED** [review]
🐢 is pondering this flavor and will get back to you soon.

File: Items/04_00_WeaponAttachments.txt [hunk 2]
```diff
@@ -350,6 +350,14 @@ more effeciently. This attachment can only be put on laser weapons.
 
     Weapon Damage:  +10% (Drop the remainder) or +5, whichever is higher.
 
+== Capacitor Rgulator $1500 ==
+
+   An addition to a laser gun that helps conserve power usage of the guns battery
```
Comment (bleehu) [inline Items/04_00_WeaponAttachments.txt:355]: apostrophe: "to the gun's battery"

File: Items/04_00_WeaponAttachments.txt [hunk 3]
```diff
@@ -350,6 +350,14 @@ more effeciently. This attachment can only be put on laser weapons.
 
     Weapon Damage:  +10% (Drop the remainder) or +5, whichever is higher.
 
+== Capacitor Rgulator $1500 ==
+
+   An addition to a laser gun that helps conserve power usage of the guns battery
+charge, through manipulation of the energy discharge of internal capacitors. By 
+spreading out the discgarge of energy in a single blast to higher accurate levels,
```
Comment (bleehu) [inline Items/04_00_WeaponAttachments.txt:357]: sp: discharge

Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt:357]: grammar: "to _more_ accurate levels"

**Comment by Turtlelord26** [comment]
By the way, the PR is based from RC,  and should probably be rebased to dev.

**Review by bleehu — CHANGES_REQUESTED** [review]
I like the concept! Some spelling and grammar fixes, and I think we should halve the price or so. Once the grammar is fixed, I'll approve it.

File: Items/04_00_WeaponAttachments.txt (hunk 1 above)
Comment (Masterdiekillsu) [reply Items/04_00_WeaponAttachments.txt:353]: lol! Thank you, also do you want me to base this out of dev instead of R_C?

Comment (bleehu) [reply Items/04_00_WeaponAttachments.txt:353]: oh geeze. Yes please.

**Comment by bleehu** [comment]
Also, don't forget to link the issue that this would solve!

**Review by Turtlelord26 — COMMENTED** [review]
🐢 is not the largest fan of this flavor. I see the argument for an effective capacity increase by increasing the efficiency of the internal power train, but I don't think I can buy that there's enough inefficiency in a top-of-the-line Rigistruct or ECSC rifle to have an effect. Would you consider instead an attachment that manages to capture waste heat from firings and convert it back into capacitor charge? Probably makes the weapon less wieldy (reducing reflex). The difference is that it increases efficiency after the shot instead of before it.

**Comment by bleehu** [comment]
This is stale and the dev has left the team. I'm killing it.

**Pull request closed without merging**